### ISSUE TITLE
Do you have any plans to extend initial functionality? 

### ISSUE THREAD

**Issue by bagobor** [decimad/luabind-deboostified#23]
Ideas: 
- serialization
- class to deal with modules/chunks 
- more c++11\14 support

**Comment by decimad** [comment]
The thing is, I find creating separate bindings ultimately a waste of time... I'm !slowly! developing a C++ reflection library (that is, it is supposed to be used even within c++ for serialization, modularization and ui development... I'd argue it would be a better idea to have the lua or whatever binding a submodule of this... That being said I'm happily accepting detailed proposals or pull requests, which helps discovering practical use cases.

**Comment by bagobor** [comment]
I totally agree. But after benchmarking I found that luabind stiff is the fastest lua binding. 
About reflection/serialization/ui - do you know about https://github.com/cpgf/cpgf ?
